features/vertical-form-controls.yml

```
@@ -0,0 +1,6 @@
+name: Vertical form controls
+description: Vertical form controls display elements in a form, such as radios, progress bars, or selects, vertically when the writing mode is `vertical-lr` or `vertical-rl`
```

In the original phrasing, the cause-and-effect was unclear to me. I think this makes it more concrete. Towards Dietrich's commentary, I think it will also help us understand what this is part of, when it's time to merge it into  `writing-mode`.

```suggestion
description: The `writing-mode` property orients form elements (such as radio buttons, progress bars, or select menus) vertically when the writing mode is `vertical-lr` or `vertical-rl`.
```